src/v2/cookbook/using-axios-to-consume-apis.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Using Axios To Consume APIs
```

Comment:
Title case wise, it should be `to` I believe.

Reply:
Nice catch :)